music_assistant/helpers/audio.py

```diff
@@ -283,7 +283,10 @@ async def get_stream_details(
             try:
                 BYPASS_THROTTLER.set(True)
                 streamdetails = await music_prov.get_stream_details(
-                    prov_media.item_id, media_item.media_type
+                    prov_media.item_id,
+                    media_item.media_type,
+                    queue_id=queue_item.queue_id,
+                    queue_item_id=queue_item.queue_item_id,
```

Comment:
please remove this leftover code, its touching a core controller in a provider PR